# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: mozglue/baseprofiler/core/shared-libraries-win32.cc
# Commit: 3ea11e90d26d
# Full Hash: 3ea11e90d26dd8974b3da41335f5861c491b0c90
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2024-09-17 09:28:38
# Regressor Bug: 1634785
# File Overlap Count: 1
# Description:
#   Bug 1634785 - Rename BaseProfilerSharedLibraries.h to SharedLibraries.h r=mstange,profiler-reviewers
#   
#   Now that we only have one implementation of this code, we don't have to
#   differentiate it with "BaseProfiler".
#   
# ==============================================================================

diff -r cb492d775d37 -r 3ea11e90d26d mozglue/baseprofiler/core/shared-libraries-win32.cc
--- a/mozglue/baseprofiler/core/shared-libraries-win32.cc	Mon Sep 16 21:09:34 2024 +0000
+++ b/mozglue/baseprofiler/core/shared-libraries-win32.cc	Mon Sep 16 21:09:34 2024 +0000
@@ -5,7 +5,7 @@
 
 #include <windows.h>
 
-#include "BaseProfilerSharedLibraries.h"
+#include "SharedLibraries.h"
 
 #include "mozilla/glue/WindowsUnicode.h"
 #include "mozilla/NativeNt.h"